gap/FinSetsForCAP.gi

```diff
@@ -158,6 +158,17 @@ InstallMethod( FinSetNC,
     
 end );
 
+##
+InstallMethod( \<,
+        "for two CAP finite sets",
+        [ IsFiniteSet, IsFiniteSet ],
+        
+  function( set1, set2 )
+    
+    return AsList( set1 ) < AsList( set2 );
```

Comment:
According to the GAP documentation, `<` must be a total order. This is only fulfilled if `IsEqualForElementOfFinSets` is given by `=` for the data types in question. A concrete example: one could decide to compare strings in finite sets via `IsIdenticalObj` (for example for performance reasons). Then for
```
A := FinSet( [ "a" ] );
B := FinSet( [ "a" ] );
```
the following would be true:
```
not (A < B) and not(B < A)
```
but
```
A = B
```
would not be false.

Could you add a warning to `IsEqualForElementOfFinSets` and check the existing installations of `IsEqualForElementOfFinSets` with regard to this problem?

Reply:
Edit: I accidentally wrote

> would not be false

instead of

> would be false

above. It is fixed now, please don't be confused by the original version :D